apps/framework-cli/src/framework/schema.rs

```diff
@@ -106,11 +105,9 @@ impl Display for DuplicateModelError {
 impl std::error::Error for DuplicateModelError {}
 
 pub fn is_schema_file(path: &Path) -> bool {
-    path.to_string_lossy().contains(SCHEMAS_DIR)
```

Comment:
IIRC, this would return true on flow files then eventually try to parse the flow.ts files. Do we need to exclude flow directory then?

Reply:
Oh, I'll add this check in the watcher code then.

Reply:
I think watcher is local only. Would it still try to parse flow.ts files in prod?

Reply:
When initializing, the dir crawling has a correct base path to start.